vertx/src/main/scala/com/ibm/csync/vertx/Main.scala

```diff
@@ -312,6 +313,7 @@ object Main extends LazyLogging {
     }
   }
 
+  //scalastyle:off
```

Comment:
Need a scalastyle: on and preferably just specify the rules you want off, not all of them.